Cargo.toml

```diff
@@ -36,6 +36,7 @@ product-name = "Space Acres"
 
 [dependencies]
 anyhow = "1.0.80"
+image = "0.25.0"
```

Comment:
This dependency is not needed on Windows and should be in corresponding platform-specific section like Windows dependencies. Also please move newly added dependencies such that everything remains sorted alphabetically.

Also for `image` we only need `png` support, so use `default-features = false, features = ["png"]`, otherwise by default it pulls support for huge number of formats we don't need: https://github.com/image-rs/image/blob/4594c80b3626076af128b7a853c8e5c5187f7a64/Cargo.toml#L70